Signal/src/call/PeerConnectionClient.swift

```diff
@@ -285,9 +359,9 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
     public func setCameraSource(useBackCamera: Bool) {
         SwiftAssertIsOnMainThread(#function)
 
-        let handle = self.handle
-        PeerConnectionClient.signalingQueue.async { [weak self] in
```

Comment:
Lastly, I removed `[weak self]` from these blocks.  They cause build warnings.  The intent was to protect us from accidentally creating strong references to `self` if we touch these blocks in the future, but the swift compiler will warn us about implicit usages of `self` anyway.  We just need to be careful to not accidentally use `self.`.